.typos.toml

```diff
@@ -26,6 +26,9 @@ extend-ignore-re = [
     # youtube embed in articles
     "\\{\\{ youtube_player\\(video_id=\"[^\"]+\"\\) \\}\\}",
 
+    # Matrix
+    "morg", # shorthand for m.org or matrix.org
```

Comment:
I think using an abbreviation for this is an anti-pattern to be avoided. It adds jargon with no additional value.